### DISCUSSION TITLE
Allow me to reintroduce the Projects and Issues Category  [Projects & Issues Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#63398] [Projects and Issues]
Issues and Projects has gotten a small facelift with the new name of the category: Projects and Issues. As a reminder, the Projects and Issues category is only for conversations related to the GitHub feature areas: Projects (for planning and tracking work on GitHub - not any open source or other type of project) and Issues (the feature that allows you to track ideas, feedback, tasks, or bugs for work on GitHub. Not for any issue you are having).

Thanks for getting this bit of housekeeping done with me… now let’s celebrate some awesome members. 

🦸‍♂️ **Incredible in Projects and Issues**: A round of applause for these community citizens 👏
* @flavioaraujozup: Shared [great feedback](https://github.com/orgs/community/discussions/61959) on the assignees field and their use case for it. Appreciate you 
* @cilerler & @hamzahc1: For their curiosity and feedback on the Issues & Projects Update [discussion post](https://github.com/orgs/community/discussions/61871). We hope to hear more from both of you! 

🐙 **Projects and Issues Info**: All updates, info, releases, etc that are Issues and Projects-related!
* **The Latest Projects and Issues Update**: Now available, board swimlanes and the ability to create issues in repository groups. [Get all the info on this update from our changelog](https://github.blog/changelog/2023-07-27-github-issues-projects-july-27th-update/). 
* **More Issues & Projects Updates x 2** : There were two more updates in July - don’t miss them. [July 13’s update](https://github.blog/changelog/2023-07-13-github-issues-projects-july-13th-update/) brought sorting improvements and more. While [July 20th’s](https://github.blog/changelog/2023-07-20-github-issues-projects-july-20th-update/) includes updates to project templates, a pinned item side panel, and pull request support in tasklists!. 

⬆️ **Level up**: Resources to take your skills to new heights ☁️
* **Finish your projects**: Don’t let fear, or that last 10% hold you back with this [ReadME Project inspiration](https://github.com/readme/guides/code-visibility-codesee) from @aarondfrancis. 
* **A checklist and guide to get your repository collaboration-ready**: In the world of software development, collaboration can make the difference between a brittle last-minute release and a reliable, maintainable, pain-free project, @peckjon share settings, content, and behavior to make your collaboration a success in his [blogpost](https://github.blog/2023-08-04-a-checklist-and-guide-to-get-your-repository-collaboration-ready/).  

With three rounds of updates containing new features in the past month? Which one(s) are you most excited to use?

**Comment by mtcA6** [comment]
Board Swimlanes is a great addition, however, the title of the swimlane is left adjusted 
![image](https://github.com/community/community/assets/129431463/04066d2f-8029-4cef-8d5b-672a355b1d4a)

so as i scroll to the right I'm unable to view it. 
![image](https://github.com/community/community/assets/129431463/acb257b8-5d85-459c-b286-fdf7b487c506)

**Reply by queenofcorgis** [reply]
@mtcA6 thanks for your patience - do you mind sharing this feedback in our [discussion](https://github.com/orgs/community/discussions/61871) that features Project Board Swimlanes? This will ensure all feedback is in one place and ensures the proper team will see it. Thank you!

**Comment by queenofcorgis** [comment]
More great feedback - join me in celebrating some awesome community citizens in our [September Projects & Issues Check-in](https://github.com/orgs/community/discussions/66385)!
